### DISCUSSION TITLE
Shouldn't default ExceptWhenOwnerHasTotalClaimBlocks be lower ?

### DISCUSSION THREAD

**Discussion by gg4goat** [GriefPrevention/GriefPrevention#1107] [Q&A] [answered]
If something ever goes wrong with the config file, many claims could be deleted by mistake.

this.config_claims_expirationExemptionTotalBlocks = config.getInt("GriefPrevention.Claims.Expiration.AllClaims.ExceptWhenOwnerHasTotalClaimBlocks", 10000);

I'd personally prefer 1000 since this is the default accrued block limit.

**Comment by Jikoo** [comment]
Realistically that isn't GP's problem. A good server admin a) doesn't make changes to configurations on a production server, b) is smart enough to look in the logs to ensure that the plugin they modified the config of enabled correctly, and c) keeps regular backups. I get where you're coming from - most server admins are not good server admins - but I believe that most people just plop in GP, up the block limits and days till claims expire, and never think about the other expiration settings at all. With the higher default for people being ineligible requiring a default of 100 hours of active play (assuming caps are high enough) instead of 10, GP is unlikely to see as many issues about claims not expiring.

All that said, I'm currently trying to push people to test [GPClaimExpiration](https://github.com/GPAddons/GPClaimExpiration) for me, and when I have enough people saying it's working (or at least enough people complaining about it not working who provide information I can fix it with) I'll be submitting a PR to GP to remove claim expiration from the core entirely.

**Comment by RoboMWM** [comment]
The default accrued claim block limit should be 80,000 iirc, not 1000?

**Comment by RoboMWM** [comment]
Interesting, not sure if expiration should be one of the first things to go, but I guess considering how many people complain/have opinions on this, it probably is a good idea to take out.

**Comment by Jikoo** [comment]
> Interesting, not sure if expiration should be one of the first things to go, but I guess considering how many people complain/have opinions on this, it probably is a good idea to take out.

Yeah, it was volume that started me working on it. There are a bunch of longstanding issues (496, 474, 260, 932, and 58 specifically are ones on my list) in addition to recurring questions/concerns that could be resolved by a rewrite. If we're gonna rewrite it, may as well go for future state and add all the bells and whistles.

> The default accrued claim block limit should be 80,000 iirc, not 1000?

Looks like it's documented as being 80k but internally is 2k: https://github.com/TechFortress/GriefPrevention/blob/8465d19d28cc6b826f7cebaeccafa52855a29e7b/src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java#L535

**Comment by RoboMWM** [comment]
Hmm, must've been changed somewhere, because I remember (and had in my config before I modified it) it at 80k

**Comment by RoboMWM** [comment]
https://github.com/TechFortress/GriefPrevention/commit/c7327d498299339f8fdb9f217d5e5d970e6705d2#diff-bd72da6eb91964fbaa474bafcaac56434a55926a9743ec41c3dba6de30eedc86L531

This is when it changed, when he added the multiple accrual rate permissions.

**Comment by gg4goat** [comment]
@Jikoo I will give a try to your GPClaimExpiration plugin, I will let you know if I find any issue.

**Comment by Jikoo** [comment, marked as answer]
Since the default has now been restored to 80k, I think we're good here - players will have the opportunity to gain more blocks than required to permanently protect claims by default.
